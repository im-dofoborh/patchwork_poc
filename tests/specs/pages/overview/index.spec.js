import { mount, createLocalVue } from '@vue/test-utils';
import VueRouter from 'vue-router';
import OverviewPage from '~/pages/overview';

describe('Overview page', () => {
  let localVue = null;
  let wrapper = null;
  let router = null;

  beforeEach(() => {
    localVue = createLocalVue();
    localVue.component('overview-page', OverviewPage);
    localVue.use(VueRouter);
    router = new VueRouter();
    wrapper = mount(OverviewPage, {
      localVue,
      router,
    });
  })

  it('renders', () => {
    expect(wrapper.html()).toMatchSnapshot();
  });
});
