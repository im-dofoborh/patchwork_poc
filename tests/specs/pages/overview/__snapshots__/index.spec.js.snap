// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Overview page renders 1`] = `
<section>
  <div class="sg-content-editor editor-content"><a href="https://github.immediate.co.uk/WCP/patchwork/edit/develop/content/overview.md" target="_blank" class="sg-content-editor__link">
      Edit on GitHub
    </a>
    <div class="mt-md">Mock content</div>
  </div>
</section>
`;
